### ISSUE TITLE
Run R machine learning model on Azure

### ISSUE THREAD

**Issue by palatiner** [stlrda/AI_Assessor#5]
Depending on the amount compute power needed for these models we might need to run them on Azure. We need somebody with experience in this area.

**Comment by bransonf** [comment]
I was thinking something along the lines of a docker container with a very simple Shiny front-end. Once we have the process fully completed, I'll create a front end for inputing any parameters if necessary, and return downloadable html reports with the results as well as graphics (addressing #9).

I have no trouble doing this, but is someone going to foot the azure bill?

**Comment by palatiner** [comment]
Yes! Since Microsoft is sponsoring this data dive I heard they can give us credits on Azure. Let me get more info.

**Comment by c-michael-o-fugue-co** [comment]
https://docs.microsoft.com/en-us/azure/machine-learning/studio/what-is-ml-studio

**Comment by mikeaorlando** [comment]
https://docs.microsoft.com/en-us/azure/machine-learning/studio/what-is-ml-studio
